### PR TITLE
RHIDP-2681: Remove unsupported auth providers

### PR DESCRIPTION
**Pull request by linfraze**
**Versions:**
1.2+

**Jira:** 
https://issues.redhat.com/browse/RHIDP-2681

**Preview:** 
https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-343/admin-rhdh/#enabling-authentication

**Reviews:**
- [ ] SME: @kim-tsao 
- [ ] QE: N/A
- [ ] Doc: N/A

### CODE REVIEW

File: assemblies/assembly-enabling-authentication.adoc [hunk 1]
```diff
@@ -11,24 +11,25 @@ Authentication providers are typically used in the following ways:
...
-Okta:: `okta`
-OAuth 2 Custom Proxy:: `oauth2Proxy`
-OneLogin:: `onelogin`
-SAML:: `saml`
-
-For each provider that you want to use, follow the dedicated procedure to:
+//GitLab:: `gitlab`
+//Google:: `google`
+//Google IAP:: `gcp-iap`
+Keycloack:: `oidc`
```
Comment (kim-tsao) [inline assemblies/assembly-enabling-authentication.adoc:26]: typo, should be `Keycloak`

Comment (linfraze) [reply assemblies/assembly-enabling-authentication.adoc:26]: Oof, good eye!

**Comment by kim-tsao** [comment]
`lgtm`
`approve`

**Comment by kim-tsao** [comment]
\lgtm
\approve

**Pull request merged**